omnibus/config/projects/chef-workstation.rb

```diff
@@ -67,17 +67,18 @@
 dependency "rust-uninstall"
 
 dependency "chef-dk"
+dependency "chef-dk-gem-versions"
 
 dependency "gem-permissions"
 dependency "rubygems-customization"
+dependency "shebang-cleanup"
 
 if windows?
   dependency "chef-dk-env-customization"
   dependency "chef-dk-powershell-scripts"
 end
 
 dependency "version-manifest"
-dependency "clean-static-libs"
```

Comment:
?

Reply:
Copied from https://github.com/chef/chef-dk/blob/v3.11.1/omnibus/config/projects/chefdk.rb#L69

Reply:
Seems like we want that but if we're not doing it in DK we should nuke it for now

Reply:
I searched for a while through a few different means and could not figure out when that line got removed. I think it got dropped during crazytown days.

Reply:
https://github.com/chef/omnibus-software/pull/1058 this cleans it up by adding the old logic into our new ruby cleanup def